README.md

```diff
@@ -87,3 +87,10 @@ You can contribute to the project on github. Here's how you can do it.
 <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5.png"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/css3.png"/> </a><a href="https://www.php.net/" target="_blank"> <img src="https://img.icons8.com/dusk/64/000000/php-logo.png"/>
 
 </br>
+
+ <hr>
+
```

Comment:
can you please add some valuable text above the Figma link?

Reply:
Added